### PR TITLE
Fix/staging bugfix and remove betablocker

### PR DESCRIPTION
**Pull request by jl24**
Fixes the three issues mentioned in #104

### CODE REVIEW

File: project/webapp/static/js/utils/locate.js [hunk 1]
```diff
@@ -173,7 +173,7 @@ cw.MapView = BB.View.extend({
             lng: place.geometry.location.lng()
         };
         this.model.set('coordinates', coordinates);
-        this.getLegislators(coordinates);
+        // this.getLegislators(coordinates);
```
Comment (brylie) [inline project/webapp/static/js/utils/locate.js]: Cleanup commented code.

File: project/webapp/static/js/utils/locate.js [hunk 2]
```diff
@@ -199,7 +199,7 @@ cw.MapView = BB.View.extend({
                 }
 
                 _this.model.set('coordinates', geolocation);
-                _this.getLegislators(geolocation);
+                // _this.getLegislators(geolocation); # Sunlight API is Deprecated
```
Comment (brylie) [inline project/webapp/static/js/utils/locate.js]: Cleanup commented code.

**Comment by brylie** [comment]
This looks good. Just clean up the commented code.

**Comment by jl24** [comment]
Removed commented code

**Pull request merged**